lib/SimplePie/SimplePie.php

```diff
@@ -33,7 +33,7 @@
  * POSSIBILITY OF SUCH DAMAGE.
  *
  * @package SimplePie
- * @version 1.5.6
+ * @version 1.5.8
  * @copyright 2004-2017 Ryan Parman, Sam Sneddon, Ryan McCue
```

Comment:
Should we mark the files that are not 100% SimplePie? When I read version 1.5.8, than I would expect, that the source code is 100% SimplePie 1.5.8 without any individual changes.

Reply:
Ok, but then it is two more lines to update every time :-)

Reply:
Why 2 lines? How would you add this information?

Reply:
I would rather not, but otherwise the convention (used by e.g. Debian, etc.) is a suffix such as `1.5.8-freshrss`.
The version information is on two different lines